### ISSUE TITLE
0.1.3 different in git than in maven?

### ISSUE THREAD

**Issue by KadekM** [lihaoyi/acyclic#10]
Hey @lihaoyi,

can you please check:
https://github.com/lihaoyi/acyclic/blob/master/src/main/scala/acyclic/plugin/PluginPhase.scala#L34-L37

Here it's ok, but when I check the ones on maven, it's different:
```
  def units = global.currentRun
                    .units
                    .filter(!_.isJava)
                    .toSeq
                    .sortBy(_.source.content.mkString.hashCode())
```
which makes java files not to compile into project... it was how I did it (screwed up) initially but then changed it to actual fix, somehow this discrepancy happened... my actual fixes got merged though

can you check it out please ? I believe 0.1.3 - maven one - is now broken.

**Comment by lihaoyi** [comment]
Ah, maybe I screwed up the publish? I'll take a look

**Comment by KadekM** [comment]
Maybe...

**Comment by lihaoyi** [comment]
The version in maven looks like the version in git to me
```
haoyi-mbp:Downloads haoyi$ unzip acyclic_2.11-0.1.3-sources.jar
Archive:  acyclic_2.11-0.1.3-sources.jar
  inflating: META-INF/MANIFEST.MF
   creating: acyclic/
   creating: acyclic/plugin/
  inflating: acyclic/plugin/PluginPhase.scala
  inflating: acyclic/plugin/DependencyExtraction.scala
  inflating: acyclic/plugin/Plugin.scala
  inflating: acyclic/package.scala
  inflating: acyclic/plugin/GraphAnalysis.scala
  inflating: scalac-plugin.xml
haoyi-mbp:Downloads haoyi$ cat acyclic/plugin/PluginPhase.scala
...
  def units = global.currentRun
                    .units
                    .toSeq
                    .sortBy(_.source.content.mkString.hashCode())
...
```

I'm not sure what problem you're describing

**Comment by KadekM** [comment]
Indeed. I messed up. Sorry. I had this bad version but it didn't come from maven but from work's nexus, where I published it under 0.1.3 just so we can use it at work until you publish it to maven.
So I went full retard.
